cmd/ipsw/cmd/dyld/dyld_disass.go

```diff
@@ -62,6 +62,8 @@ func init() {
 	DisassCmd.Flags().Bool("dec-nocache", false, "Do not use decompilation cache")
 	DisassCmd.Flags().Float64("dec-temp", 0.2, "LLM temperature for decompilation")
 	DisassCmd.Flags().Float64("dec-top-p", 0.1, "LLM top_p for decompilation")
+	DisassCmd.Flags().Int("dec-retries", 0, "Number of retries for LLM decompilation")
+	DisassCmd.Flags().Float64("dec-retry-backoff", 30.0, "Backoff time in seconds between retries")
```

Comment:
this should be a Duration 

see: https://github.com/blacktop/ipsw/blob/10534629ea77e881124d6ed2dcbbdfbf4d5e9473/cmd/ipsw/cmd/watch.go#L79